kidviz/views.py

```diff
@@ -997,3 +1001,12 @@ def get(self, request, *args, **kwargs):
 
         messages.add_message(self.request, messages.SUCCESS, 'Draft Dismissed.')
         return HttpResponseRedirect(reverse_lazy('observation_view'))
+
+
+class WorkQueue(LoginRequiredMixin, ListView):
+    """View used to display work queue."""
+    template_name = 'work_queue.html'
+    paginate_by = 10
+
+    def get_queryset(self):
+        return Observation.objects.filter(owner=self.request.user, constructs=None)
```

Comment:
In template you query for all students for every observation. To minimize number of queries use `prefetch`.